### PR TITLE
fix: Update evaluate_rag_rapid_evaluation_sdk.ipynb to GenAI Eval SDK

### PR DESCRIPTION
**Pull request by lakeyk**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [x] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [x] You are listed as the author in your notebook or README file.
  - [ ] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [x] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [x] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb [hunk 1]
```diff
@@ -25,7 +25,7 @@
    "cell_type": "markdown",
```
Comment (jsondai) [inline gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <h1>"Gen AI",  throughout</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/986/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <p>Remove second line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/986/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>if not PROJECT_ID or PROJECT_ID == "[your-project-id]":</p><p>    raise ValueError("Please set your PROJECT_ID")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/986/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (jsondai) [inline gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <p>Line  7 &amp; 8 can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/986/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (jsondai) [reply gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <p>Sorry, let's keep line 8.</p>

Comment (jsondai) [inline gemini/evaluation/evaluate_rag_rapid_evaluation_sdk.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #2.    nest_asyncio.apply()
</pre><p>This line can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/986/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Review by holtskinner — APPROVED** [review]

**Pull request merged**